packages/ui-kit-theme/README.md

```diff
@@ -0,0 +1,33 @@
+# UI Kit Theme
+
+This package contains light and dark themes from the UI Kit Design System. To learn more visit the project's:
+
+- [Storybook](https://adaptiveconsulting.github.io/ui-kit-2022/)
+- [GitHub](https://github.com/AdaptiveConsulting/ui-kit-2022)
+
+## Installation
+
+Run the following command to install the package using [NPM](https://www.npmjs.com/)
+
+```
+npm i @ui-kit-2022/theme
+```
+
+## Usage
+
+To apply a theme import it from the `@ui-kit-2022/theme` package and use a ThemeProvider:
+
+```
+import { ThemeProvider } from "@mui/material/styles";
+import { light, dark } from "@ui-kit-2022/theme";
+import CssBaseline from "@mui/material/CssBaseline";
+
```

Comment:
I think you'll want to include that we want to import the CSS file with the fonts.

At least showing it in the code should be enough:

`import '@ui-kit-2022/theme/dist/style.css';`

Reply:
That's a great suggestion and something I overlooked.  I've added the import line to the example.